src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random getRandom = new Random();
+
+    public static Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setBallColor(ColorSupplier.getRandomColor());
+        randomBall.setBallNumber(getRandom.nextInt(100));
```

Comment:
[Create constant variable](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)